djangocms_blog/search_indexes.py

```diff
@@ -47,32 +47,32 @@ def get_model(self):
         return Post
 
     def get_search_data(self, post, language, request):
-        optional_attributes = []
-        abstract = post.safe_translation_getter('abstract')
+        description = post.get_description()
+        keywords = post.get_keywords()
+
         text_bits = [post.get_title()]
-        text_bits.append(strip_tags(abstract))
-        text_bits.append(post.get_description())
```

Comment:
get_desription utilizes abstract if there is no description so I guess it's better to replace description+abstract to get_description